lms/djangoapps/onboarding_survey/models.py

```diff
@@ -188,12 +221,100 @@ class UserInfoSurvey(models.Model):
+    org_capacity_leadership = models.BooleanField(blank=True, default=False)
+    # PartnerNetwork
+    partner_network = models.ForeignKey(
+        PartnerNetwork, on_delete=models.SET_NULL, related_name='user_history', blank=True, null=True
+    )
+    # PersonalGoal
+    personal_goal_gain_new_skill = models.BooleanField(blank=True, default=False)
+    personal_goal_build_relation_with_other = models.BooleanField(blank=True, default=False)
+    personal_goal_develop_leadership = models.BooleanField(blank=True, default=False)
+    personal_goal_improve_job_prospect = models.BooleanField(blank=True, default=False)
+    personal_goal_contribute_to_organization = models.BooleanField(blank=True, default=False)
+    # RoleInsideOrg
+    role_in_org = models.ForeignKey(
+        RoleInsideOrg, on_delete=models.SET_NULL, related_name='user_history', blank=True, null=True
+    )
+    # TotalEmployees
+    org_total_employees = models.ForeignKey(
+        TotalEmployee, on_delete=models.SET_NULL, related_name='user_history', blank=True, null=True
+    )
+    # TotalVolunteers
+    org_total_volunteers = models.ForeignKey(
+        TotalVolunteer, on_delete=models.SET_NULL, related_name='user_history', blank=True, null=True
+    )
+    # UserInfoSurvey
+    dob = models.DateField(blank=True, null=True)
+    language = models.CharField(max_length=255)
+    country_of_residence = models.CharField(max_length=255)
+    city_of_residence = models.CharField(max_length=255, blank=True)
+    is_employment_location_different = models.BooleanField(default=False)
+    country_of_employment = models.CharField(max_length=255, blank=True)
+    city_of_employment = models.CharField(max_length=255, blank=True)
+    # InterestSurvey
+    reason_of_selected_interest = models.CharField(max_length=255, blank=True)
+    # OrganizationSurvey
+    org_start_month_year = models.CharField(max_length=100, blank=True)
+    org_country = models.CharField(max_length=255)
+    org_city = models.CharField(max_length=255, blank=True)
+    org_url = models.URLField(max_length=255, blank=True)
+    org_founding_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    org_total_annual_clients_or_beneficiary = models.PositiveIntegerField(blank=True, null=True)
+    org_total_annual_revenue_for_last_fiscal_year = models.CharField(max_length=255, blank=True)
+    # Registration
+    POC_CHOICES = ((0, 'No'), (1, 'Yes'))
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    organization = models.ForeignKey(
+        Organization, related_name='user_history', blank=True, null=True, on_delete=models.SET_NULL
+    )
+    is_poc = models.BooleanField(choices=POC_CHOICES, default=False)
+    is_currently_employed = models.BooleanField(default=False)
+    org_admin_email = models.EmailField(blank=True, null=True)
+
+    start_date = models.DateTimeField(auto_now_add=True)
```

Comment:
I think we should change `start_date` and `end_date` to `created_at` and `modified_at`

Reply:
These fields names were suggested by client (Josh, who would be using this table for Analytics).